__mocks__/fs-extra-promise.js

```diff
@@ -75,6 +75,28 @@ function copyDirSync(src, dest) {
   }
 }
 
+/**
+ * Utility function to copy a directory to a destination recursively
+ * copyDirAsync needs to use fs.copyAsync as behaviour of copyAsync and copyFileSync differ
```

Comment:
This comment is unnecessary as outside of the commit, the context for this comment is missing.

Reply:
Makes sense, will remove the comment then!